xboxone/run.sh

```diff
@@ -3,6 +3,8 @@
 # Copy over the xbox component
 mkdir -p /config/custom_components/xboxone
 cp -f xboxone.py /config/custom_components/xboxone/media_player.py
+cp -f __init__.py /config/custom_components/xboxone/__init__.py
+cp -f manifest.json /config/custom_components/xboxone/manifest.json
```

Comment:
Thanks for the PR! Could you just collapse the `cp` commands into one? 

```bash
cp -f xboxone.py __init__.py manifest.json /config/custom_components/xboxone/
```

Reply:
Sorry it was late. Yes that makes more sense. I've made the change but unable to test at the moment

Reply:
I'll have time to verify tonight as well 👍

Reply:
So, something unexpected is happening. I'm getting the following errors in the logs:

```
[services.d] starting services
[services.d] done.
cp: can't stat '__init__.py': No such file or directory
cp: can't stat 'manifest.json': No such file or directory
[2019-08-14 22:23:59,615] INFO in rest_server: Starting Xbox Smartglass REST server started on 0.0.0.0:5557
```

This seems to be because of the Dockerfile, which doesn't import everything in the repo's `xboxone` directory, and instead explicitly only brings in `run.sh` and `xboxone.py`. We'd have to bring the manifest and init files in too.

However, as you can see from the log, the server ended up starting just fine. I'm guessing this is because I previously had a working installation, so the manifest information might already be cached somewhere... so I think it's just me.

Reply:
I'll open a separate thread for the `Dockerfile` change.